### ISSUE TITLE
Fish prompt in GNOME terminal

### ISSUE THREAD

**Issue by hastebrot** [oh-my-fish/theme-default#5]
The fish prompt is displayed incorrectly using the GNOME terminal on Ubuntu 14.04. I also tried the Powerline fonts, but the same issue.

![screenshot from 2016-01-12 16 02 20](https://cloud.githubusercontent.com/assets/496255/12266962/2a38f7f0-b946-11e5-89c0-cd4eaf10a910.png)

**Comment by bobthecow** [comment]
Are you referring to the shape of the fish at the start of the line? If so, that's not powerline-related, it's due to the glyph your font uses.

**Comment by hastebrot** [comment]
Ah, I see. Yes, the fish in the fish prompt. I changed it to "⨯>". Not exactly a fish, but looks better with the font I use than the previous anamorphic chars.

Closing this.

**Comment by bobthecow** [comment]
Yeah, unfortunately this theme is using those characters as design elements rather than characters, so it doesn't work equally well for all fonts.

**Comment by frederickjh** [comment]
I know that this issue got closed but I just wanted to say that I too am seeing this on GNOME terminal on Ubuntu 14.04. I have the font set to Ubuntu Mono. I tried many sans font but I never found one that did not overlap the two characters. I always had to have a space between them that they did not overlap. I tried a ZERO WIDTH SPACE U+200B (ZWSP), but they overlapped with this too. I was able to use a HAIR SPACE U+200A, which it says is the thinnest space available, but this still leave a gap between the two characters.
